File: ios/NativeModule.m
// RCTCalendarModule.m
#import <Foundation/Foundation.h>
#import <React/RCTBridgeModule.h>
#import <React/RCTEventEmitter.h>

@interface RCT_EXTERN_MODULE(NativeModule, NSObject)

RCT_EXTERN_METHOD(log: (nonnull NSString *)level message: (nonnull NSString *)message)
RCT_EXTERN_METHOD(sumTwoNumbers: (nonnull NSNumber *)a b: (nonnull NSNumber *)b)

@end

File: ios/NativeModule.swift
//
//  LoggingModule.swift
//  MentorshipMarch2022
//
//  Created by Matt Bowen on 3/9/22.
//

import Foundation

@objc(NativeModule)
class NativeModule: NSObject, RCTBridgeModule {
  static func moduleName() -> String! {
    return "NativeModule"
  }
  
  @objc func log(_ level: String, message: String) {
    print(">>>>> From RN \(message)")
  }
  
  @objc func sumTwoNumbers(_ a: NSNumber, b: NSNumber) {
    print(Int(a) + Int(b))
  }
}
